roles/letsencrypt/tasks/certificates.yml

```diff
@@ -22,7 +22,7 @@
   changed_when: false
   when: site_uses_letsencrypt
   with_dict: "{{ wordpress_sites }}"
-  tags: [wordpress, wordpress-setup, nginx-includes]
+  tags: [wordpress, wordpress-setup, nginx-includes, nginx-sites]
```

Comment:
Question: Why this task needs those tags?

Reply:
If a user runs `server.yml` with `--tags <any tag above>`, Trellis may template Nginx confs that use the `letsencrypt_cert_ids` variable (e.g., in [this PR's `ssl.no-default.conf.j2`](https://github.com/roots/trellis/pull/888/files#diff-9d628e80c8099ce78eaac01fd61000afR28) or in [`wordpress-site.conf.j2`](https://github.com/roots/trellis/blob/ec9324ccbc1910115d382d0559784824084d683f/roles/wordpress-setup/templates/wordpress-site.conf.j2#L93)). The [`letsencrypt_cert_ids`](`https://github.com/roots/trellis/blob/ec9324ccbc1910115d382d0559784824084d683f/roles/letsencrypt/defaults/main.yml#L4`) variable definition relies on the above task's registered var `generate_cert_ids`. If the task above does not also run for the `tags` above, `letsencrypt_cert_ids` will be malformed and the Nginx conf templating tasks will fail.

Reply:
Thanks!